code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -237,12 +237,12 @@
 
 			if ((H.getToxLoss() > toxThreshHold) && (amount>0)) // If toxloss is above a certain threshhold, more toxin damage will cause internal organ damage. For reference: 50 is DANGEROUS TOXIN LEVELS DETECTED
 				var/obj/item/organ/internal/targeted_organ
-				if ((L.is_usable()) || (H.getToxLoss() > 100)) // If the liver is dead, the toxin begins to rot out the other organs. If you have more than 100 toxin, your body begins to rot regardless of liver status
+				if ((!L.is_usable()) || (H.getToxLoss() > 100)) // If the liver is dead, the toxin begins to rot out the other organs. If you have more than 100 toxin, your body begins to rot regardless of liver status
```

Comment:
you sure you want this?

Reply:
95% yes?

The logic is (If LIVER IS NOT usable OR total toxin damage >100) then target something other than the liver. Else target the liver.